setup.py

```diff
@@ -0,0 +1,24 @@
+from time import time
+
+from setuptools import setup
+from setuptools_scm.version import ScmVersion
+
+
+def get_dev_timestamp(version: ScmVersion) -> str:
+    """Returns a new distribution version string.
+
+    Returns the version found in the git tag if currently checked out commit has a tag.
+    Otherwise, returns the version found in the latest previous git tag, appended with `dev` + current timestamp
```

Comment:
```suggestion
    Otherwise, returns the version found in the most recent git tag, appended with
    `dev` and the current timestamp in seconds.
```